### ISSUE TITLE
Accessibility: Alternative text for the image is a file name

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#429]
**Description**
This looks to be happening with items that are uploaded within pages that are not Items within the exhibit. Images that are uploaded in this way have their filename added as alternative text, which is not accessible. I have noticed that all the pages with this issue seem to have added images using the Horizontal Row widget.

[Arachnophilia example](https://my2.siteimprove.com/Inspector/602695/Accessibility/Page?pageId=44986392377&impmd=3CC33A985F387DC37EA87D3D5A2306C4#/Criterion/1.1.1/Check/26)

This issue is only being flagged on some images but is a problem on all of them that are being added with the Horizontal Row template. The ones that are not being flagged are because the file names have spaces in them, making them appear as though they aren't actually file names, but are instead legitimate alternative text.

**Solution**
Option 1: Require that images used on pages are uploaded as an Item within each exhibit, which would have a box for alternative text from #420.
Option 2: In the Horizontal Row, add a box for each item to be able to add alternative text and have the template read from that instead of the image filename.
